lib/swagger-client.js

```diff
@@ -2689,10 +2708,10 @@ SwaggerOperation.prototype.urlify = function (args) {
           addition += encodeURIComponent(param.name) + '=' + encodeURIComponent(args[param.name]);
         } else {
           if (param.required)
-            throw "" + param.name + " is a required query param.";
+            throw '' + param.name + ' is a required query param.';
```

Comment:
this was introduced in bc8f0f68b5de91b8fc26d2433909be010d2b61a7, again not part of my change but I can change it if you want.